src/App.js

```diff
@@ -1,10 +1,15 @@
 import React from 'react';
+import { Route } from 'react-router-dom';
 import './App.css';
 import { Home, About, Contact, Navigation } from './components';
 
+
 const App = () => (
   <div>
     <Navigation />
+    <Route exact path='/' component={Home} />
```

Comment:
Can you reason why we need an `exact` here? Type your answer as a comment in this thread.